.gitattributes

```diff
@@ -0,0 +1 @@
+./Codegen/Generated/* linguist-generated=true
```

Comment:
I think the `./` might be messing this up.

```suggestion
Codegen/Generated/* linguist-generated=true
```

Reply:
I think that is correct yes.